core/src/main/java/cz/fi/muni/pv217/employee/management/client/VacationRESTClient.java

```diff
@@ -0,0 +1,17 @@
+package cz.fi.muni.pv217.employee.management.client;
+
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import java.time.LocalDate;
+
+@RegisterRestClient(configKey = "vacation-service")
+public interface VacationRESTClient {
```

Comment:
What is this used for?

Reply:
It will be used to communicate with vacation service. I need information about employee vacation when creating workday.